### PR TITLE
Servlet

### PR DESCRIPTION
**Pull request by Kaisar283**
@Alexmansar Hi
I've finished homework-4, please check.
But I couldn't connect Open API to the project.

### CODE REVIEW

File: src/main/java/kz/andersen/java_intensive_13/exception/ExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package kz.andersen.java_intensive_13.exception;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+public class ExceptionHandler {
```
Comment (Alexmansar) [inline src/main/java/kz/andersen/java_intensive_13/exception/ExceptionHandler.java]: should not be here. I mean in this package. Just define other one please

File: src/main/java/kz/andersen/java_intensive_13/models/Apartment.java [hunk 2]
```diff
@@ -77,4 +92,16 @@ public boolean equals(Object o) {
...
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        Apartment apartment = new Apartment(1, 4000);
+
+        String json = objectMapper.writeValueAsString(apartment);
+        System.out.println(json);
+
+        Apartment apartment1 = objectMapper.readValue(json, Apartment.class);
+        System.out.println(apartment1);
+    }
```
Comment (Alexmansar) [inline src/main/java/kz/andersen/java_intensive_13/models/Apartment.java]: its correct??

File: src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java [hunk 3]
```diff
@@ -265,12 +267,12 @@ public void exitCommand() {
...
-        int apartment2 = apartmentService.registerApartment(3000);
-        int apartment3 = apartmentService.registerApartment(2000);
-        int apartment4 = apartmentService.registerApartment(4000);
-        apartmentService.reserveApartment(apartment2, alice);
+//        int apartment1 = apartmentService.registerApartment(1000);
+//        int apartment2 = apartmentService.registerApartment(3000);
+//        int apartment3 = apartmentService.registerApartment(2000);
+//        int apartment4 = apartmentService.registerApartment(4000);
+//        apartmentService.reserveApartment(apartment2, alice);
     }
```
Comment (Alexmansar) [inline src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java]: why this ode is commented?

File: src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java [hunk 4]
```diff
@@ -16,6 +17,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Disabled
```
Comment (Alexmansar) [inline src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java]: and this is disabled?

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@Kaisar283 Hi) please, check the comments

**Comment by Alexmansar** [comment]
> @Alexmansar Hi I've finished homework-4, please check. But I couldn't connect Open API to the project.

why you want to use it?

File: src/main/java/kz/andersen/java_intensive_13/models/Apartment.java (hunk 2 above)
Comment (Kaisar283) [reply src/main/java/kz/andersen/java_intensive_13/models/Apartment.java]: No, forgot delete it

File: src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java (hunk 4 above)
Comment (Kaisar283) [reply src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java]: tests are disabled because the UserConsole class no longer in use

**Comment by Kaisar283** [comment]
In the previews lesson I saw that Open API you have used and I thought we should use too

File: src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java (hunk 4 above)
Comment (Kaisar283) [reply src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java]: Can I delete UserConsole class in the project, and tests too?

Comment (Alexmansar) [reply src/test/java/kz/andersen/java_intensive_13/console/UserConsoleTest.java]: You van leave. Just add comment why tests are disabled)

**Comment by Alexmansar** [comment]
> In the previews lesson I saw that Open API you have used and I thought we should use too

not required))

**Review by Alexmansar — APPROVED** [review]
@Kaisar283 Could be merged)

**Pull request merged**